signwriting_animation/diffusion/core/models.py

```diff
@@ -126,7 +128,11 @@ def forward(self,
         xseq = self.sequence_pos_encoder(xseq)
         output = self.seqEncoder(xseq)[-num_frames:]
         output = self.pose_projection(output)
-        return output
+        global_latent = self.global_norm(xseq.mean(0))
+        print("Global latent stats: min", global_latent.min().item(), "max", global_latent.max().item()) 
+
+        length_dist = self.length_predictor(global_latent)
+        return output, length_dist
```

Comment:
Would be good to update the docstring of the `forward` function to show the type, description and shape of the additional output variable